### PR TITLE
Feat/lending data attr

### PR DESCRIPTION
**Pull request by rick23p**
https://taiga.sovryn.app/project/sovryn-light/task/695
https://taiga.sovryn.app/project/sovryn-light/task/698
https://taiga.sovryn.app/project/sovryn-light/task/699

### CODE REVIEW

File: src/app/pages/BorrowPage/BorrowingContainer/index.tsx [hunk 1]
```diff
@@ -221,6 +222,7 @@ const InnerBorrowContainer: React.FC<Props> = ({ currency }) => {
                   onChange={item => setTokenToCollarate(item.key)}
                   value={tokenToCollarate}
                   items={collaterals}
+                  dataActionId="borrow-assetToCollatelizeAmountDropDown"
```
Comment (soulBit) [inline src/app/pages/BorrowPage/BorrowingContainer/index.tsx:225]:
```suggestion
                  dataActionId="borrow-assetToCollateralizeAmountDropDown"
```

File: src/app/pages/LendingPage/components/CurrencyContainer/CurrencyRow.tsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ const CurrencyRow: React.FC<Props> = ({
             className="tw-block tw-w-full tw-mb-3 tw-rounded-lg tw-bg-primary-25 hover:tw-opacity-75"
             textClassName="tw-text-base"
             disabled={depositLocked || !connected}
+            data-action-id={`lend-deposit-button-${asset}`}
```
Comment (soulBit) [inline src/app/pages/LendingPage/components/CurrencyContainer/CurrencyRow.tsx:58]:
```suggestion
            data-action-id={`lend-depositButton-${asset}`}
```
This was defined incorrectly on the spec

File: src/app/pages/LendingPage/components/CurrencyContainer/CurrencyRow.tsx [hunk 3]
```diff
@@ -81,6 +82,7 @@ const CurrencyRow: React.FC<Props> = ({
             className="tw-block tw-w-full tw-rounded-lg"
             textClassName="tw-text-base"
             disabled={isEmptyBalance || withdrawLocked || !connected}
+            data-action-id={`lend-withdrawal-button-${asset}`}
```
Comment (soulBit) [inline src/app/pages/LendingPage/components/CurrencyContainer/CurrencyRow.tsx:85]:
```suggestion
            data-action-id={`lend-withdrawalButton-${asset}`}
```
This was defined incorrectly on the spec

File: src/app/pages/LiquidityMining/components/AddLiquidityDialog/AddLiquidityDialogV1.tsx [hunk 4]
```diff
@@ -170,6 +171,7 @@ export const AddLiquidityDialogV1: React.FC<IAddLiquidityDialogV1Props> = ({
                 tx.loading || !valid || !canInteract || addliquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-deposit-button"
```
Comment (soulBit) [inline src/app/pages/LiquidityMining/components/AddLiquidityDialog/AddLiquidityDialogV1.tsx:174]:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```

Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals

File: src/app/pages/LiquidityMining/components/AddLiquidityDialog/index.tsx [hunk 5]
```diff
@@ -133,6 +134,7 @@ export const AddLiquidityDialog: React.FC<IAddLiquidityDialogProps> = ({
                 tx.loading || !valid || !canInteract || addliquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-deposit-button"
```
Comment (soulBit) [inline src/app/pages/LiquidityMining/components/AddLiquidityDialog/index.tsx:137]:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```
Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals

File: src/app/pages/LiquidityMining/components/MiningPool/index.tsx [hunk 6]
```diff
@@ -71,13 +71,15 @@ export const MiningPool: React.FC<IMiningPoolProps> = ({
           className="tw-block tw-w-full tw-mb-3 tw-rounded-lg tw-bg-primary-25 hover:tw-opacity-75"
           textClassName="tw-text-base"
           disabled={!canInteract || addliquidityLocked}
+          data-action-id={`yieldFarm-deposit-button-${pool.key}`}
```
Comment (soulBit) [inline src/app/pages/LiquidityMining/components/MiningPool/index.tsx:74]:
```suggestion
          data-action-id={`yieldFarm-depositButton-${pool.key}`}
```

File: src/app/pages/LiquidityMining/components/MiningPool/index.tsx [hunk 7]
```diff
@@ -71,13 +71,15 @@ export const MiningPool: React.FC<IMiningPoolProps> = ({
...
         />
         <ActionButton
           text={t(translations.liquidityMining.withdraw)}
           onClick={() => setDialog(DialogType.REMOVE)}
           className="tw-block tw-w-full tw-rounded-lg"
           textClassName="tw-text-base"
           disabled={!canInteract || isEmptyBalance || removeliquidityLocked}
+          data-action-id={`yieldFarm-withdrawal-button-${pool.key}`}
```
Comment (soulBit) [inline src/app/pages/LiquidityMining/components/MiningPool/index.tsx:82]:
```suggestion
          data-action-id={`yieldFarm-withdrawalButton-${pool.key}`}
```

File: src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/RemoveLiquidityDialogV1.tsx [hunk 8]
```diff
@@ -242,6 +243,7 @@ export const RemoveLiquidityDialogV1: React.FC<IRemoveLiquidityDialogV1Props> =
                 tx.loading || !valid || !canInteract || removeliquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-withdrawal-button"
```
Comment (soulBit) [inline src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/RemoveLiquidityDialogV1.tsx:246]:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```

Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals

File: src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/index.tsx [hunk 9]
```diff
@@ -183,6 +184,7 @@ export const RemoveLiquidityDialog: React.FC<IRemoveLiquidityDialogProps> = ({
                 tx.loading || !valid || !canInteract || removeLiquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-withdrawal-button"
```
Comment (soulBit) [inline src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/index.tsx:187]:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```

Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals

**Review by soulBit — CHANGES_REQUESTED** [review]
Few small changes to make based on some errors in the spec

**Review by tiltom — CHANGES_REQUESTED** [review]
The code itself looks ok but I was inspecting screens and the Borrow page (https://taiga.sovryn.app/project/sovryn-light/task/698) is not done according to the spec - I was only able to find `borrow-amountInput` attribute, all the others are missing.

**Review by tiltom — APPROVED** [review]

**Review by soulBit — APPROVED** [review]

**Review by JoeStorm99 — APPROVED** [review]

**Pull request merged**